katas/langtonant/lorena/ActionAnt.js

```diff
@@ -0,0 +1,45 @@
+// Box color of grid
+
+export function colorBox (x, y, colorAct) {
+    var color = 0;
+    if (colorAct == 0) {
+        color = 1;
+    } else {
+        color = 0;
+    }
+    return color;
+}
+
+
+// Ant´s Orientation
+
+export function newOri (x, y, ori, colorAct) {
```

Comment:
in some cases, try to use complete word for name, for example `newOrientation`, so ambiguity is avoided (say, this could be also read as `newOrigin`?)